.drone.star

```diff
@@ -3156,3 +3164,63 @@ def pipelineSanityChecks(ctx, pipelines):
 
     for image in images.keys():
         print(" %sx\t%s" % (images[image], image))
+
+def uploadErrorTrace():
+    return [{
+        "name": "upload-trace",
+        "image": "plugins/s3",
+        "pull": "if-not-exists",
+        "settings": {
+            "bucket": "owncloud",
+            "endpoint": {
+                "from_secret": "cache_s3_endpoint",
+            },
+            "path_style": True,
+            "source": "%s/reports/e2e/playwright/tracing/**/*" % dir["web"],
+            "strip_prefix": "%s/reports/e2e/playwright/tracing" % dir["web"],
+            "target": "/web/tracing/${DRONE_BUILD_NUMBER}",
+        },
+        "environment": {
+            "AWS_ACCESS_KEY_ID": {
+                "from_secret": "cache_s3_access_key",
+            },
+            "AWS_SECRET_ACCESS_KEY": {
+                "from_secret": "cache_s3_secret_key",
+            },
+        },
+        "when": {
+            "status": [
+                "failure",
+            ],
+            "event": [
+                "pull_request",
+            ],
+        },
+    }]
+
+def buildGithubCommentForE2ETests(suite):
```

Comment:
naming sounds confusing for me, can we rename it to something like `publishTracingResult`